mail_environment/README.rst

```diff
@@ -0,0 +1,117 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
use .PNG image:
`.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png`

Reply:
svg is better for 2 reasons: it's lighter and size can be adapted ;)